PWGJE/FlavourJetTasks/AliAnalysisTaskHFJetIPQA.h

```diff
@@ -298,7 +297,7 @@ class AliAnalysisTaskHFJetIPQA: public AliAnalysisTaskEmcalJet
     }
     Bool_t IsParton(int pdg);
     Bool_t IsParticleInCone(const AliVParticle* part, const AliEmcalJet* jet, Double_t dRMax);
-    Int_t NDaughterInCone(AliAODMCParticle* posdaugh, const AliAODMCParticle* negdaugh, const AliEmcalJet* jet, const AliAODEvent* event, Double_t dRMax, Double_t& ipsig);
+    Int_t NDaughterInCone(vector<Int_t>& vecDaughLabels, const AliEmcalJet* jet, const AliAODEvent* event, Double_t dRMax, Double_t& ipsig);
```

Comment:
Please replace all remaining occurrences of vector with std::vector. The "using namespace std" comes in implicitly from AliJetShapes.h and will be removed in the near future - it is super dangerous.